i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/users/README.md

```diff
@@ -166,4 +166,46 @@ Logto 默认使用 [Argon2](https://en.wikipedia.org/wiki/Argon2) 的实现 [nod
 
 ### 自定义数据（custom data）
 
+_custom data_ 即自定义数据，可以在这个可自定义的属性中保存额外的用户信息。
```

Comment:
i think we need to rephrase.

> _custom data_ contains additional info in customizable attributes.
> 可以在这个可自定义的属性中保存额外的用户信息。

these two have different meanings, while they are not informative too

Reply:
Updated the English part to 

```markdown
`custom_data` (hereinafter called Custom Data) contains additional info in this customizable attribute.
```

Reply:
Remove the description here and only keep the description at the beginning of the Profile section above.